### PR TITLE
[CI:DOCS] Fix typos and improve language

### PR DESCRIPTION
**Pull request by eriksjolund**
Signed-off-by: Erik Sjölund <erik.sjolund@gmail.com>

#### What type of PR is this?

> /kind api-change
> /kind bug
> /kind cleanup
> /kind deprecation
> /kind design
> /kind documentation
> /kind failing-test 
> /kind feature
> /kind flake
> /kind other

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

**Comment by eriksjolund** [comment]
Maybe I shouldn't have added the `[CI:DOCS]` because of the modification in _cmd/buildah/source.go_ ...

File: docs/Containerfile.5.md [hunk 1]
```diff
@@ -216,7 +216,7 @@ The secret needs to be passed to the build using the --secret flag. The final im
   If you use the shell form of the **CMD**, the `<command>` executes in `/bin/sh -c`:
 
   Note that the exec form is parsed as a JSON array, which means that you must
-  use double-quotes (") around words not single-quotes (').
+  use double-quotes (") around words, not single-quotes (').
```
Comment (TomSweeneyRedHat) [inline docs/Containerfile.5.md:219]: Just noting that https://github.com/containers/buildah/pull/3746 and https://github.com/containers/common/pull/905 are moving this file from Buildah to c/common.  We should ensure these changes move too.  @Foxboron FYI

File: docs/buildah-build.1.md [hunk 2]
```diff
@@ -688,14 +688,14 @@ Only the current container can use a private volume.
...
-  Any subsequent execution of `RUN` commands sees the original source directory content, any changes from previous RUN commands no longer exists.
+  Any subsequent execution of `RUN` commands sees the original source directory content, any changes from previous RUN commands no longer exist.
 
   One use case of the `overlay` mount is sharing the package cache from the host into the container to allow speeding up builds.
 
   Note:
 
      - The `O` flag is not allowed to be specified with the `Z` or `z` flags. Content mounted into the container is labeled with the private label.
-       On SELinux systems, labels in the source directory needs to be readable by the container label. If not, SELinux container separation must be disabled for the container to work.
+       On SELinux systems, labels in the source directory need to be readable by the container label. If not, SELinux container separation must be disabled for the container to work.
```
Comment (TomSweeneyRedHat) [inline docs/buildah-build.1.md:698]: Suggestion for consideration

```suggestion
       On SELinux systems, labels in the source directory must be readable by the container label. If not, SELinux container separation must be disabled for the container to work.
```

**Comment by TomSweeneyRedHat** [comment]
Awesome changes @eriksjolund TYVM!  I'm really shocked by the "using using" bit in so many places.
One comment for consideration, then you may need to have a merge dance with @Foxboron for the containerfile related files.

File: docs/Containerfile.5.md (hunk 1 above)
Comment (Foxboron) [reply docs/Containerfile.5.md:219]: I can produce a very ad-hoc patch and apply on top of my branch so @eriksjolund get author credit there :)

**Comment by Foxboron** [comment]
@TomSweeneyRedHat Feel free to merge these first. I'll do a rebase and include a patch with these changes (where relevant) on top of my change :) Thanks!

**Comment by eriksjolund** [comment]
Is there anything I should do?
The extra commit 
https://github.com/containers/buildah/pull/3745/commits/d92da92e4d4b476beac000d2f2b4932c80811cae
looks good but it is marked with a red cross. Please guide me.

**Comment by rhatdan** [comment]
Could you pull both and merge them

**Comment by rhatdan** [comment]
I decided to force merge.

**Comment by eriksjolund** [comment]
@rhatdan Thanks for merging

**Pull request merged**